src/helpers/helpers.options.js

```diff
@@ -65,6 +65,22 @@ module.exports = {
 		};
 	},
 
+	parseFontOptions: function(options, globalDefaults) {
+		var valueOrDefault = helpers.valueOrDefault;
+		var size = valueOrDefault(options.fontSize, globalDefaults.defaultFontSize);
+		var style = valueOrDefault(options.fontStyle, globalDefaults.defaultFontStyle);
+		var family = valueOrDefault(options.fontFamily, globalDefaults.defaultFontFamily);
+		var lineHeight = valueOrDefault(options.lineHeight, globalDefaults.defaultlineHeight);
+
+		return {
+			size: size,
+			style: style,
+			family: family,
+			font: helpers.fontString(size, style, family),
```

Comment:
What do you think of renaming `font` to `string` (as in the datalabels plugin)? I think `font.string` is easier to understand and more explicit than `font.font` when `var font = _parseFont(...);`